hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationServiceV1.java

```diff
@@ -110,6 +111,30 @@
         registerJavaTypeSerializers();
     }
 
+    @Override
+    public <B extends Data> B toData(Object obj, DataType type) {
```

Comment:
I don't have a strong reason to keep it this way, I just thought it would be simpler than having two EE event journal classes overriding some `toOnHeapData()` method and instantiating those classes in EE code. What do you think? Is there a better way of getting on-heap `Data` in OS?